integration-tests/cli/product-export.it.js

```diff
@@ -0,0 +1,117 @@
+import fs from 'fs'
+import tmp from 'tmp'
+import { getCredentials } from '@commercetools/get-credentials'
+import { exec } from 'child_process'
+import { version } from '@commercetools/product-exporter/package.json'
+import {
+  sampleProductType,
+  sampleTaxCategory,
+  createProducts,
+  expectedProducts,
+} from './helpers/products-export.data'
+import { createData, clearData } from './helpers/utils'
+
+let projectKey
+if (process.env.CI === 'true')
+  projectKey = 'product-export-integration-test'
+else
+  projectKey = process.env.npm_config_projectkey
```

Comment:
wouldn't this be `npm_config_projectKey`?

Reply:
I think npm lowercase it's arguments.

Reply:
It's all lower case :-)